app/controllers/image_uploads_controller.rb

```diff
@@ -42,19 +45,12 @@ def cloudinary_link(uploaders)
 
   private
 
-  def image_upload(images)
-    if images.is_a? Array
-      images.map do |image|
-        uploader = ArticleImageUploader.new
+  def upload_images(images, rate_limiter)
+    Array.wrap(images).map do |image|
+      ArticleImageUploader.new.tap do |uploader|
```

Comment:
[tap](https://rubyapi.org/2.6/o/object#method-i-tap) always returns the upload at the end of the block after all of the contained operations

Reply:
That is so cool.